Makefile

```diff
@@ -1,12 +1,42 @@
-# Run formatters and linters
-.PONY: lint
+.PHONY: help lint build-frontend build start stop update
+
+COMPOSE_FILE = infra/docker/docker-compose.yml
+
+.DEFAULT_GOAL := help
+
+help:
+	@echo "Available commands:"
+	@echo "  make lint              - Run formatters and linters"
+	@echo "  make build-frontent    - Build frontend"
+	@echo "  make build             - Build Docker image"
+	@echo "  make start             - Start container"
+	@echo "  make stop              - Stop container"
+	@echo "  make update            - Update to latest version"
+
 lint:
 	isort --settings-path=./pyproject.toml --recursive .
 	black --config=./pyproject.toml .
 	flake8 .
 	mypy ./voc_builder ./tests
 
-.PONY: build-frontend
 build-frontend:
 	rm -rf voc_builder/notepad/dist
-	cd voc_frontend && VITE_AIVOC_API_ENDPOINT='' npm run build-only && mv dist ../voc_builder/notepad
\ No newline at end of file
+	cd voc_frontend && VITE_AIVOC_API_ENDPOINT='' npm run build-only && mv dist ../voc_builder/notepad
+
+build:
+	docker-compose -f $(COMPOSE_FILE) build --no-cache
+
+start:
```

Comment:
Most of the commands are not directly related to the development of the application, so I would prefer not to see them in the Makefile.